docs/source/guides/ToolModding/tools.rst

```diff
@@ -37,6 +37,10 @@ File editors
 * `Visual Studio Code <https://code.visualstudio.com>`_
 * `VSCodium <https://vscodium.com>`_
 * `Notepad++ <https://notepad-plus-plus.org/downloads/>`_
+  
+* `Kate <https://kate-editor.org/>`_
+
+  * `Kate syntax highlighting for Squirrel <https://gist.github.com/CTalvio/6de535f9258cfebd71ab64d7e6af4ee6>`_
```

Comment:
I'm curious why GitHub diff shows stuff like this even though it's already merged... Wtf?

Reply:
Is this branch maybe somehow not up-to-date with main?

Reply:
um yes the issue was that I placed it in a different file

Reply:
But it should still see it as just moving the file then though, no?

This is actually kinda annoying cause it bloats the div, hmmm

Reply:
no like, where the link was put, was in a totally different file
I just moved the whole section from one file to another